models/tranquil/tranquil_txs.sql

```diff
@@ -0,0 +1,131 @@
+{{
+    config(
+        materialized='table',
+        unique_key='log_id',
+        tags=['core', 'defi', 'amm', 'lending']
+        )
+}}
+
+with
+
+tranquil_market_labels as (
+    select
+        *
+    from {{ ref('tranquil_market_labels') }}
+),
+
+-- Tranquil contracts all have the same "Creator" or admin address
+tranquil_contracts as (
+    select distinct
+        evm_contract_address
+    from logs 
+    where event_inputs:admin::string = '0x15424ab0bbab79bad32ce779197748485b5ae456'
+), 
+
+tranquil_markets_usd as (  
+    select
+        *
+    from {{ ref('tranquil_markets_tokenprice') }}
+),
+
+tranquil_deposits as (
+    select 
+        logs.log_id,
+        logs.tx_hash,
+        block_timestamp,
+        block_id,
+        event_inputs:minter::string as user_address,
+        logs.evm_contract_address,
+        tml.token_symbol,
+        (event_inputs:mintAmount::number) as token_amount_raw,
+        (event_inputs:mintAmount::number / pow(10, tml.decimals)) as token_amount,
+        'Deposit' as tx_type
+    from logs 
+    inner join tranquil_contracts as tc 
+        on logs.evm_contract_address = tc.evm_contract_address
+    left join tranquil_market_labels as tml
+        on logs.evm_contract_address = tml.token_address
+    where logs.event_name = 'Mint'
+),
+
+tranquil_borrows as (
+    select 
+        logs.log_id,
+        logs.tx_hash,
+        block_timestamp,
+        block_id,
+        event_inputs:borrower::string as user_address,
+        logs.evm_contract_address,
+        tml.token_symbol,
+        (event_inputs:borrowAmount::number) as token_amount_raw,
+        (event_inputs:borrowAmount::number / pow(10, tml.decimals)) as token_amount,
+        'Borrow' as tx_type
+    from logs 
+    inner join tranquil_contracts as tc 
+        on logs.evm_contract_address = tc.evm_contract_address
+    left join tranquil_market_labels as tml
+        on logs.evm_contract_address = tml.token_address
+    where logs.event_name = 'Borrow'
+),
```

Comment:
can this and the following be all combined in one with event_name including all mint, borrow etc, and use case when for `tx_type`?